Building-ChakraCore.md

```diff
@@ -45,32 +45,46 @@ msbuild ... /p:RuntimeLib=static_library ...
 
 ## Linux ##
 
-Steps below are tested on Ubuntu 16.04LTS. In order to compile ChakraCore on other Linux distrubitions,
-use distro's package manager instead of `apt-get` to install given dependencies below.
+Steps below are tested on Ubuntu 16.04LTS and Fedora 24. In order to compile ChakraCore on other Linux distrubitions,
+use distro's package manager instead of `apt-get` / `dnf` to install given dependencies below.
 
 To build ChakraCore on Linux: (requires Clang 3.7+)
 
 * Open terminal and start the update process.
```

Comment:
Can remove this line as well as it was meant to describe the `apt-get update; apt-get dist-upgrade` steps which are included below.